TestResultSummaryService/routes/getFeedbackUrl.js

```diff
@@ -0,0 +1,13 @@
+module.exports = async ( req, res ) => {
+    const { repoName, buildName, issueName, issueCreator, accuracy } = req.query;
+    let feedbackUrlRes = {};
+
+    feedbackUrlRes.repoName = repoName;
+    feedbackUrlRes.buildName = buildName;
+    feedbackUrlRes.issueName = issueName;
+    feedbackUrlRes.issueCreator = issueCreator;
+    feedbackUrlRes.accuracy = accuracy;
+
+    console.log(feedbackUrlRes);
+    res.send( { output: feedbackUrlRes } );
```

Comment:
Since your API function now works to receive data, you can add checks for these data, then send back "error" or "success", instead of sending the data back again, refer to https://github.com/adoptium/aqa-test-tools/blob/7b6f85476ea6a3bbd66cb7d6d07b68ba85414147/TestResultSummaryService/routes/compareTestDuration.js#L30